core/src/main/java/io/grpc/DecompressorRegistry.java

```diff
@@ -54,7 +54,7 @@
 public final class DecompressorRegistry {
 
   private static final DecompressorRegistry DEFAULT_INSTANCE = new DecompressorRegistry(
-      new DecompressorInfo(new Codec.Gzip(), false),
+      new DecompressorInfo(new Codec.Gzip(), true),
```

Comment:
Prior to this PR, there wasn't a good way to turn it off, so it wasn't advertised.  This PR adds the relevant controls so its okay to turn it on.